rust/coverage.stats

```diff
@@ -1 +1 @@
-91.7
\ No newline at end of file
+90.0
```

Comment:
Is it something wanted?

Reply:
Actual code coverage is 91.3% on Linux CI, but for some reasons locally I have 90.9%
I have still not found the reason, I noticed the coverage mismatch on this PR: https://github.com/trustwallet/wallet-core/pull/3350
Other than that, I'd be happy to increase the code coverage, but that will take some time to find proper test cases.